.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,11 +2,11 @@
 
 ## Description
```

Comment:
where the emojis 😢

Reply:
There's never been emojis in ours...

Reply:
its proven that emojis in PR template improve developer productivity

Reply:
Citation?

Reply:
(Lin et al, 2019)

Reply:
Wow, you're published? Congrats man! I think we should _definitely_ announce this at the next LP meeting 😛